psalm.baseline.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<files psalm-version="3.11.5@3c60609c218d4d4b3b257728b8089094e5c6c6c2">
+<files psalm-version="3.14.2@3538fe1955d47f6ee926c0769d71af6db08aa488">
   <file src="src/Client.php">
-    <NullArgument occurrences="1">
-      <code>null</code>
-    </NullArgument>
     <PossiblyUndefinedVariable occurrences="1">
       <code>$options</code>
     </PossiblyUndefinedVariable>
+    <UndefinedConstant occurrences="1">
+      <code>\IDNA_DEFAULT</code>
```

Comment:
The new docker image seems to mean we loose all static analysis of extensions not installed in the image... Not a problem, for undefined constants, but really unfortunately for all the function calls that are no longer analyzed...

Reply:
Which function calls?

Reply:
All functions from extensions not installed in the image are no longer have their uses type-checked. Like ext-intl.

Reply:
I only find this: https://github.com/guzzle/guzzle/blob/879a6c664fda9d33f274d7a9b3cd65ac2f1b62a6/src/Utils.php#L389

Reply:
Right, but I suppose my point was what other extensions are being skipped now, that we didn't notice?